src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -21,15 +23,21 @@ object ModelTypes {
 
 object ClusterStatus extends Enumeration {
   type ClusterStatus = Value
-  val Unknown, Creating, Deleting, Deleted = Value
-  val activeStatuses = Seq(Unknown, Creating)
+  val Unknown, Creating, Running, Updating, Error, Deleting, Deleted = Value
```

Comment:
It doesn't look like `Updating` is used anywhere

Reply:
Dataproc has a status Updating, I tried to map that dataproc statuses in Leo. If a Cluster is Updating we'll keep monitoring it until it's Running.

Reply:
(We check for Updating in ClusterMonitorActor)

Reply:
ok